test/prow/testcases/common/cron_dataload.py

```diff
@@ -0,0 +1,177 @@
+                return False
+            if opRef.get("DataLoad", "") != dataload_name:
+                print("DataLoad {} is running but dataset opRef {}".format(dataload_name, opRef))
+                return False
+        time.sleep(1)
+
+    return True
+
+def main():
+    if os.getenv("KUBERNETES_SERVICE_HOST") is None:
+        config.load_kube_config()
+    else:
+        config.load_incluster_config()
+
+    name = "hbase"
+    namespace = "default"
+
+    dataset = fluidapi.assemble_dataset("alluxio-webufs") \
+        .set_namespaced_name(namespace, name) \
+        
+    runtime = fluidapi.assemble_runtime("alluxio-webufs") \
+        .set_namespaced_name(namespace, name) \
+        .set_tieredstore(mediumtype="MEM", path="/dev/shm", quota="2Gi")
+    
+    dataload_name = "jindo-dataload"
+    cron_dataload = fluidapi.DataLoad(name=dataload_name, namespace=namespace) \
+        .set_target_dataset(name, namespace) \
+        .set_load_metadata(True) \
+        .set_cron("* * * * *")
+    
+    flow = TestFlow("Common - Test Cron Dataload")
+
+    flow.append_step(
+        SimpleStep(
+            step_name="create dataset",
+            forth_fn=funcs.create_dataset_fn(dataset.dump()),
+            back_fn=funcs.delete_dataset_and_runtime_fn(runtime.dump(), name, namespace)
+        )
+    )
+
+    flow.append_step(
+        SimpleStep(
+            step_name="create runtime",
+            forth_fn=funcs.create_runtime_fn(runtime.dump()),
+            back_fn=dummy_back
+        )
+    )
+
+    flow.append_step(
+        StatusCheckStep(
+            step_name="check dataset bound",
+            forth_fn=funcs.check_dataset_bound_fn(name, namespace)
+        )
+    )
+
+    flow.append_step(
+        StatusCheckStep(
+            step_name="check if PV & PVC is ready",
+            forth_fn=funcs.check_volume_resource_ready_fn(name, namespace)
+        )
+    )
+
+    flow.append_step(
+        SimpleStep(
+            step_name="create dataload",
+            forth_fn=funcs.create_dataload_fn(cron_dataload.dump()),
+            back_fn=dummy_back,  # DataLoad should have ownerReference of Dataset
+        )
+    )
+
+    flow.append_step(
+        StatusCheckStep(
+            step_name="check if cron dataLoad status correct",
+            forth_fn=currying_fn(check_cron_dataload, dataload_name=dataload_name, dataset_name=name, namespace=namespace),
+            back_fn=dummy_back,
+        )
+    )
+
+    flow.append_step(
+        StatusCheckStep(
+            step_name="check if dataload job completes",
+            forth_fn=funcs.check_dataload_job_status_fn(dataload_name, namespace),
+        )
+    )
+
+    flow.append_step(
+        StatusCheckStep(
+            step_name="check if the whole dataset is warmed up",
+            forth_fn=funcs.check_dataset_cached_percentage_fn(name, namespace)
+        )
+    )
+
+    flow.append_step(
```

Comment:
Please check the generated cronjob. Thanks.

Reply:
ok.